largest_palindrome_product.js

```diff
@@ -10,7 +10,41 @@ module.exports = function(digits){
   var factor_1 = 0;
   var palindromeNumber = 0;
 
+  let palFactor1;
+  let palFactor2;
   // do your work here
+  if (digits === 2){
```

Comment:
Insert new line above this.

Reply:
Also, this is not scalable. This wouldn't handle any other case than digits is 2 or 3.